config/convolvedFluxes.py

```diff
@@ -0,0 +1,9 @@
+# Enable measurement of convolved fluxes
+# 'config' is a SourceMeasurementConfig
+try:
+    import lsst.meas.extensions.convolved  # noqa: Load flux.convolved algorithm
+except ImportError as exc:
+    print("Cannot import lsst.meas.extensions.convolved (%s): disabling convolved flux measurements" % (exc,))
```

Comment:
Can this be a more proper log message, or does that not work in config files?

Reply:
Changed to:
```python
except ImportError as exc:
    import logging
    logging.getLogger("lsst.obs.decam.config").warning("Cannot import lsst.meas.extensions.convolved (%s):"
                                                       " disabling convolved flux measurements", exc)
```